internal/provider/models/destinations/test/gcp_cloud_monitoring_test.go

```diff
@@ -0,0 +1,185 @@
+						description = "test dest description"
+						inputs = [mezmo_http_source.test_source.id]
+						pipeline_id = mezmo_pipeline.test_parent.id
+						resource_type = "global"
+						project_id = "proj123"
+						credentials_json = "{}"
+						resource_labels = {
+							"somekey" = "someval"
+						}
+					}`,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestMatchResourceAttr(
+						"mezmo_gcp_cloud_monitoring_destination.my_dest", "id", regexp.MustCompile(`[\w-]{36}`)),
+					StateHasExpectedValues("mezmo_gcp_cloud_monitoring_destination.my_dest", map[string]any{
+						"pipeline_id":             "#mezmo_pipeline.test_parent.id",
+						"title":                   "test dest",
+						"description":             "test dest description",
+						"generation_id":           "0",
+						"ack_enabled":             "true",
+						"inputs.#":                "1",
+						"resource_type":           "global",
+						"project_id":              "proj123",
+						"credentials_json":        "{}",
+						"resource_labels.%":       "1",
+						"resource_labels.somekey": "someval",
+					}),
+				),
+			},
+			// Import
+			{
+				Config: GetCachedConfig(cacheKey) + `
+					resource "mezmo_gcp_cloud_monitoring_destination" "import_target" {
+						title = "test dest"
+						description = "test dest description"
+						inputs = [mezmo_http_source.test_source.id]
+						pipeline_id = mezmo_pipeline.test_parent.id
+						resource_type = "global"
+						project_id = "proj123"
+						credentials_json = "{}"
+						resource_labels = {
+							somekey = "someval"
+						}
+					}`,
+				ImportState:       true,
+				ResourceName:      "mezmo_gcp_cloud_monitoring_destination.import_target",
+				ImportStateIdFunc: ComputeImportId("mezmo_gcp_cloud_monitoring_destination.my_dest"),
+				ImportStateVerify: true,
+			},
+			// update
+			{
+				Config: GetCachedConfig(cacheKey) + `
+					resource "mezmo_http_source" "new_source" {
+						pipeline_id = mezmo_pipeline.test_parent.id
+					}
+					resource "mezmo_gcp_cloud_monitoring_destination" "my_dest" {
+						title = "new test dest"
+						description = "this is a new test description"
+						inputs = [mezmo_http_source.new_source.id]
+						pipeline_id = mezmo_pipeline.test_parent.id
+						resource_type = "global2"
+						project_id = "proj456"
+						credentials_json = "{}"
+					}`,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestMatchResourceAttr(
+						"mezmo_gcp_cloud_monitoring_destination.my_dest", "id", regexp.MustCompile(`[\w-]{36}`)),
+					StateHasExpectedValues("mezmo_gcp_cloud_monitoring_destination.my_dest", map[string]any{
+						"pipeline_id":       "#mezmo_pipeline.test_parent.id",
+						"title":             "new test dest",
+						"description":       "this is a new test description",
+						"generation_id":     "1",
+						"ack_enabled":       "true",
+						"inputs.#":          "1",
+						"resource_type":     "global2",
+						"resource_labels.%": nil,
```

Comment:
This should be fine, but FYI, after I wrote this I needed a similar check.  `TestCheckNoResourceAttr` is designed for this.  If you were to use it for this property, it'd be its own assertion:
```
resource.TestCheckNoResourceAttr("mezmo_gcp_cloud_monitoring_destination.my_dest", "resource_labels")
```